<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>rtqt</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rtqt.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabCAN">
       <attribute name="title">
        <string>&amp;CAN测试</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="labelCANDev">
            <property name="text">
             <string>CAN卡设备号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboDevIndex">
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCANRate">
            <property name="text">
             <string>波特率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboCANRate">
            <property name="currentIndex">
             <number>5</number>
            </property>
            <property name="maxVisibleItems">
             <number>10</number>
            </property>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1000</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushCANOpen">
            <property name="text">
             <string>打开</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushCANClose">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>关闭</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupChannel_1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>通道1</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="labelSendData_1">
               <property name="text">
                <string>发送数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineSendData_1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelRecvData_1">
               <property name="text">
                <string>接收数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineRecvData_1"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushCANSend1">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="labelSendNum_1">
               <property name="text">
                <string>发包个数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdSendNum_1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelLostRate_1">
               <property name="text">
                <string>丢包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdLostRate_1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelErrRate_1">
               <property name="text">
                <string>错包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdErrRate_1"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupChannel_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>通道2</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="labelSendData_2">
               <property name="text">
                <string>发送数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineSendData_2"/>
             </item>
             <item>
              <widget class="QLabel" name="labelRecvData_2">
               <property name="text">
                <string>接收数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineRecvData_2"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushCANSend2">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labelSendNum_2">
               <property name="text">
                <string>发包个数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdSendNum_2"/>
             </item>
             <item>
              <widget class="QLabel" name="labelLostRate_2">
               <property name="text">
                <string>丢包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdLostRate_2"/>
             </item>
             <item>
              <widget class="QLabel" name="labelErrRate_2">
               <property name="text">
                <string>错包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdErrRate_2"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCOM">
       <attribute name="title">
        <string>&amp;COM测试</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QGroupBox" name="groupSetting">
          <property name="title">
           <string>设置</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="labelCOM1">
               <property name="text">
                <string>串口1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboCOMName1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOM2">
               <property name="text">
                <string>串口2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboCOMName2"/>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pushCOMOpen">
               <property name="text">
                <string>打开</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushCOMClose">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>关闭</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="labelRate">
               <property name="text">
                <string>波特率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboRate">
               <item>
                <property name="text">
                 <string>110</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>300</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>14400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>56000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>57600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>115200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>128000</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>256000</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelDataBit">
               <property name="text">
                <string>数据位</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboDataBit">
               <item>
                <property name="text">
                 <string>4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelStopBit">
               <property name="text">
                <string>停止位</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboStopBit">
               <item>
                <property name="text">
                 <string>One</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Onehalf</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Two</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelParity">
               <property name="text">
                <string>校验方式</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboParity">
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Even</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Odd</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Mark</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Space</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupCOM1">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>串口1</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="labelCOMSendData1">
               <property name="text">
                <string>发送数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineCOMSendData1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOMRecvData1">
               <property name="text">
                <string>接收数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineCOMRecvData1"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushCOMSend1">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="labelCOMSendNum1">
               <property name="text">
                <string>发包个数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdCOMSendNum1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOMLostRate1">
               <property name="text">
                <string>丢包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdCOMLostRate1"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOMErrRate1">
               <property name="text">
                <string>错包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdCOMErrRate1"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupCOM2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>串口2</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_10">
             <item>
              <widget class="QLabel" name="labelCOMSendData2">
               <property name="text">
                <string>发送数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineCOMSendData2"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOMRecvData2">
               <property name="text">
                <string>接收数据</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineCOMRecvData2"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushCOMSend2">
               <property name="text">
                <string>发送</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <widget class="QLabel" name="labelCOMSendNum2">
               <property name="text">
                <string>发包个数</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdCOMSendNum2"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOMLostRate2">
               <property name="text">
                <string>丢包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdCOMLostRate2"/>
             </item>
             <item>
              <widget class="QLabel" name="labelCOMErrRate2">
               <property name="text">
                <string>错包率</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLCDNumber" name="lcdCOMErrRate2"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutSoftware"/>
    <addaction name="actionAboutCompany"/>
   </widget>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAboutSoftware">
   <property name="text">
    <string>Software</string>
   </property>
  </action>
  <action name="actionAboutCompany">
   <property name="text">
    <string>Company</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rtqt.qrc"/>
 </resources>
 <connections/>
</ui>
